src/inputs/ObjectSearch.tsx

```diff
@@ -190,12 +190,17 @@ class ObjectSearch extends Component<IObjectSearchProps> {
   }
 
   private renderOption (option: IEndpointOption) {
-    const { renderOption, renderSelected } = this.fieldConfig
-       , className = `${CX_PREFIX_SEARCH_CREATE}-item`;
+    const {
+        editProps : { isOptionDisabled },
```

Comment:
editProps get passed right into the component, so you can add `isOptionDisabled?:` to `IObjectSearchProps` and  `{ IObjectSearchProps } = this.props`